### DISCUSSION TITLE
UCB CS61C: Great Ideas in Computer Architecture - csdiy.wiki

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#656] [Announcements]
# UCB CS61C: Great Ideas in Computer Architecture - csdiy.wiki

CS自学指南

https://csdiy.wiki/en/%E4%BD%93%E7%B3%BB%E7%BB%93%E6%9E%84/CS61C/

**Comment by tawaliou** [comment]
Hi I hope you’re doing well.
I’m from Benin.
I want to sincerely thank you for this docs you created which is full of really well write written courses.

Currently I’m doing the MIT 6.1810 Fall2025 Course on Operating System (they updated the 6.080 to this new one).
Big thanks.

But I have a question, 
I’ve already taken the Nand2Tetris course on Coursera last year and I’m planning maybe after the course on OS by MIT, I’ll do the course on Computer Architecture by UC Berkeley (you commented as one of the best you ever follow).
I want to level up in computer architecture, but I remark that the Berkeley course on Computer Architecture ask login information as student or guest to the university before accessing the materials, how can I do it ? Or do you’ve an idea ?
